### PR TITLE
Typos/flow changes

### PR DESCRIPTION
**Pull request by widdowquinn**
Pretty much everything is minor - flow changes are largely removal of 'the'/'their' and tense agreement, where this makes sense.

### CODE REVIEW

File: riboSeed_NAR.tex [hunk 1]
```diff
@@ -1,476 +1,476 @@
...
+
+
+The riboSeed algorithm proceeds from two observations: first, that although repeated rRNA coding sequences within a single genome are nearly identical, their flanking regions (that is, the neighboring locations within the genome) are distinct in that genome, and second, that the genomic contexts of equivalent rDNA sequences are also conserved within a taxonomic grouping. riboSeed uses only reads that map to rDNA regions from a reference genome, and is not affected by chromosomal rearrangements that occur outside the flanking regions immediately adjacent to each rRNA.
+
+
+Briefly, riboSeed uses rDNA regions from a closely-related organism's genome to help generate rDNA cluster-specific ``pseudocontigs'' derived only from the input short reads, that are seeded into the raw short reads to generate a final assembly. We refer to this process in this work as \textit{de fere novo} (meaning ``starting from almost nothing'') assembly.
```
Comment (nickp60) [inline riboSeed_NAR.tex:130]: Not sure about the word "only".

File: riboSeed_NAR.tex [hunk 2]
```diff
@@ -1,476 +1,476 @@
...
+\\
+\textbf{\texttt{select}}\\
+\texttt{select} infers ribosomal operon structure from the genomic location of constituent 16S, 23S and 5S sequences. Jenks Natural Breaks algorithm is used to group rRNA annotations into likely operons on the basis of genomic coordinates, using the number of 16S annotations to set the number of breaks. Output defines individual rDNA clusters and describes component elements in a plain text file. This output can be manually adjusted before assembly if clustering does not reflect the known arrangement of operons, for example based on visualization of the annotations in a genome browser.
+
+\subsection*{2. \textit{De Fere Novo} Assembly}
+
+\textbf{\texttt{seed}}\\
+\texttt{seed} implements the algorithm described in Figure S1. Short reads for the sequenced isolate are mapped to the reference genome using BWA\cite{Li2009}. Reads that map to each annotated rDNA and its flanking regions (where the flanking regions consist o
```
Comment (nickp60) [inline riboSeed_NAR.tex]: is this line needed?  Or should it be listed above?  All the commands are run like this

File: riboSeed_NAR.tex (hunk 1 above)
Comment (widdowquinn) [reply riboSeed_NAR.tex:130]: Does any other sequence data go into the final assembly? If not, then 'only' is appropriate, and could help fend off a question about whether we're just 'taping across' the gap with sequence from a completely sequenced reference.

File: riboSeed_NAR.tex (hunk 2 above)
Comment (widdowquinn) [reply riboSeed_NAR.tex]: Do you mean line 196? If so then yes - that line should probably go back to around line 154 or so.

I'll make the change and commit.

File: riboSeed_NAR.tex (hunk 1 above)
Comment (nickp60) [reply riboSeed_NAR.tex:130]: as per discussion, lets leave it as is!

**Review by nickp60 — APPROVED** [review]

**Pull request merged**